SwiftSnake/SwiftSnake/SKNodes/GridBackgroundNode.swift

```diff
@@ -15,12 +15,31 @@ class GridBackgroundNode: SKNode {
     private let columns = 16 * 2
     private let rows = 9 * 2
     
-    static func backgroundNode() -> GridBackgroundNode {
-        let node = GridBackgroundNode()
-        node.name = NodeType.Background.rawValue
-        return node
+    let backingStore: GridStorage<PieceToDisplay>
```

Comment:
since columns and rows are already defined at this point, could you create the backing store here instead of in init (and possibly avoid overriding initWithCoder?)